jenkinsapi_tests/unittests/test_build_scm_git.py

```diff
@@ -3,11 +3,18 @@
 from . import configs
 from jenkinsapi.build import Build
 from jenkinsapi.job import Job
+from jenkinsapi.jenkins import Jenkins
 
 
 @pytest.fixture(scope='function')
-def jenkins(mocker):
-    return mocker.MagicMock()
+def jenkins(monkeypatch):
```

Comment:
You've just created new method in conftest.py, why repeating it here and in other tests?